### PR TITLE
Rename Redis input read_timestamp to event.created

### PR DESCRIPTION
**Pull request by ruflin**
To follow ECS, read_timestamp was renamed to `event.created` for the Redis slowlog input.

### CODE REVIEW

File: dev-tools/ecs-migration.yml [hunk 1]
```diff
@@ -587,6 +587,11 @@
   to: http.version
   alias: true
 
+# Filebeat Redis Input
+
+- from: read_timestamp
+  to: event.created
+  alias: false
```
Comment (webmat) [inline dev-tools/ecs-migration.yml]: Why alias false? I think we used to use `read_timestamp` in a few places. That could be a 1:1 mapping that catches most cases.

File: filebeat/input/redis/harvester.go [hunk 2]
```diff
@@ -155,7 +155,9 @@ func (h *Harvester) Run() error {
 				"redis": common.MapStr{
 					"slowlog": subEvent,
 				},
-				"read_timestamp": common.Time(time.Now().UTC()),
+				"event": common.MapStr{
+					"created": common.Time(time.Now().UTC()),
```
Comment (urso) [inline filebeat/input/redis/harvester.go]: I think we don't need `common.Time` anymore.

File: dev-tools/ecs-migration.yml (hunk 1 above)
Comment (ruflin) [reply dev-tools/ecs-migration.yml]: For now it's false as there are still a few modules which use `read_timestamp` as a field. As soon as we completed the migration, we can probably make it an alias. Will add a checkbox to our migration issue.

File: filebeat/input/redis/harvester.go (hunk 2 above)
Comment (ruflin) [reply filebeat/input/redis/harvester.go]: You mean just using `time.Now().UTC()` instead? Did something change recently that we don't need it anymore? We use `common.Time` still quite a lot in the code base.

Comment (ruflin) [reply filebeat/input/redis/harvester.go]: Thanks for the details. +1 on removing all the current usage.

I removed common.Time and UTC conversion in the most recent commit.

**Pull request merged**